src/hssm/distribution_utils/onnx/onnx.py

```diff
@@ -280,14 +477,14 @@ def make_pytensor_logp(
 
     def logp(data: np.ndarray, *dist_params: list[float | ArrayLike]) -> ArrayLike:
         # Specify input layer of MLP
-        data = data.reshape((-1, 2))
-        inputs = pt.zeros((data.shape[0], len(dist_params) + 2))
+        data = data.reshape((-1, data_dim)) if data_dim > 1 else data
```

Comment:
don't get the purpose of this reshape

Reply:
This handles a corner case when there is only one trial and `rt, response` is provided as a 1d vector